InnerEye/ML/SSL/datamodules_and_datasets/cxr_datasets.py

```diff
@@ -175,3 +176,39 @@ def _prepare_dataset(self) -> None:
         self.dataset_dataframe.Path = self.dataset_dataframe.Path.apply(lambda x: x[strip_n:])
         self.indices = np.arange(len(self.dataset_dataframe))
         self.filenames = [self.root / p for p in self.dataset_dataframe.Path.values]
+
+
+class CovidDataset(InnerEyeCXRDatasetWithReturnIndex):
+    """
+    Dataset class to load CovidDataset dataset as datamodule for monitoring SSL training quality directly on
+    CovidDataset data.
+    We use CVX03 against CVX12 as proxy task.
+    """
+
+    def _prepare_dataset(self) -> None:
+        self.dataset_dataframe = pd.read_csv(self.root / "dataset.csv")
+        mapping = {"0": 0, "3": 0, "1": 1, "2": 1}
+        self.dataset_dataframe["final_label"] = self.dataset_dataframe.final_label.astype(int).astype(str).apply(
```

Comment:
why do we cast to `int` and back to `str`? could we not keep it as `int` and have `mapping: Dict[int, int]` or `List[int]`? might also be worth adding a comment explaining why casting is needed (e.g. is something wrong with the values read from the file?).

Reply:
I think everything is converted to string when we load the dataset within InnerEye actually. Maybe the conversion is not needed anymore indeed, would need to check

Reply:
fixed